README.md

```diff
@@ -438,7 +501,7 @@ CodeGrade! You now know how to work on and submit assignments going forward:
 - Click the **Fork** button on the Canvas assignment
 - Once the assignment is forked, clone it down to your local machine
```

Comment:
I think we should add a step after this one for running npm install and npm test. Also add the periods :)